config/values/finance.yml

```diff
@@ -1,19 +1,19 @@
 finance:
   adultdependantsgrant:
     value: £3,438
-    incomethreshold: £15,621.98
-  ## implemented above this line
+    income: £15,621.98
   childcaregrantweeklypayment:
-    onechild: £193.62
-    twoormorechildren: £331.95
+    one: £193.62
+    twoplus: £331.95
   disabilityadditionalfunding: £26,948
-  maintenanceloanpaymentvalue:
-    liveathomewithparents: £8,610
-    awayfromparentsoutsidelondon: £10,227
-    awayfromparentsinsidelondon: £13,348
+  maintenanceloan:
+    withparents: £8,610
+    notwithparentsoutsidelondon: £10,227
+    notwithparentsinsidelondon: £13,348
     oversixty: £4,327
+    ## implemented above this line
   parentslearningallowance:
-    lowpayment: £50
-    maxpayment: £1,963
-    incomethreshold: £18,835.98
-  statementofcomparabilitycost: £49.50 plus VAT
\ No newline at end of file
+    min: £50
+    max: £1,963
+    income: £18,835.98
+  statementofcomparabilitycost: £49.50 + VAT
```

Comment:
@Sarah-DfE I think this is in the wrong config file

This is the cost of a statement of comparability (ie to do with equivalent qualifications) - it is not to do with student finance

I think we should move it to the non-uk config file?

Reply:
If easier though, we could leave on this PR and move as part of a tidy-up when all the other PRs have gone out?

Reply:
Agree, and good spot!